dotnet nuget verify outputs content hash (#47528)
dotnet/docs: docs/core/tools/dotnet-nuget-verify.md

## Description

The `dotnet nuget verify` command verifies a signed NuGet package.

> From .NET 10 it also outputs the package's content hash, which may be useful to investigate lock file validation errors.

Reviewer: ```suggestion
In .NET 10 and later versions, the command also outputs the package's content hash, which might be useful to investigate lock file validation errors.
```
